doc/src/tutorial/next.rst

```diff
@@ -0,0 +1,10 @@
+What’s Next
+===========
+
+Congratulations on finishing the SymPy tutorial!
+
+The next step to help you get started is to go to the :ref:`How-to Guides <guides>` for instructions on how to do different key developer tasks.
```

Comment:
Maybe this edit or something similar?
```
If you are interested in helping to develop SymPy a good next step are the :ref:`How-to Guides <guides>` which discuss key developer tasks.
```

Reply:
Thanks for the review @smichr. How about this, to clarify that the guides are for people interested in using SymPy in their development (rather than contributing to SymPy):

```suggestion
If you are a developer interested in using SymPy in your code, please visit the :ref:`How-to Guides <guides>` which discuss key developer tasks.
```